### PR TITLE
Bind cache_status::pieces.

### PR DESCRIPTION
**Pull request by AllSeeingEyeTolledEweSew**
Per your suggestion to use `session_handle::get_cache_info`, I'd like to get python bindings for the pieces field.

### CODE REVIEW

**Comment by arvidn** [comment]
looks good. would you mind adding a call to ``test.py`` to get one of these and print it out too? just to get a bit more coverage.

**Comment by arvidn** [comment]
did you see this? https://ci.appveyor.com/project/arvidn/libtorrent/build/1.0.4410/job/bvl7q40tgb4s0rdj#L862

File: bindings/python/src/session.cpp [hunk 1]
```diff
@@ -465,6 +466,28 @@ namespace
        return ret;
     }
 
+    list cached_piece_info_list(std::vector<cached_piece_info> const& v)
+    {
+       list pieces;
+       ptime now = time_now();
```
Comment (arvidn) [inline bindings/python/src/session.cpp]: I believe this is the problem. the ``ptime`` typedef is deprecated, you should use ``lt::time_point`` instead.

**Comment by AllSeeingEyeTolledEweSew** [comment]
Sorry for the delay, I was out of town for a few days. Updated the PR per your advice. I think the CI failures are unrelated to my change.

**Pull request merged**